src/Microsoft.AspNetCore.JsonPatch/JsonPatchDocumentOfT.cs

```diff
@@ -717,7 +717,13 @@ IList<Operation> IJsonPatchDocument.GetOperations()
 
         private string GetPath<TProp>(Expression<Func<TModel, TProp>> expr)
         {
-            return "/" + GetPath(expr.Body, true).ToLowerInvariant();
+            var path = GetPath(expr.Body, true);
+            if (!String.IsNullOrEmpty(path))
+            {
+                path = "/" + path;
+            }
+
+            return path.ToLowerInvariant();
```

Comment:
Should this change just go into the `case ExpressionType.Parameter` block?

Without some examples I'm a little skeptical that this fixes all of the cases we care about.

Reply:
Can we discuss this? I am not sure how I can preemptively remove the extra slash in the `case ExpressionType.Parameter` block?